lib/services/DeleteItemFromBasket.rb

```diff
@@ -0,0 +1,22 @@
+require_relative './FetchBasketItems'
+
+module Shop
+  class DeleteItemFromBasket
+    def call(item_id)
+      basket_item = FindBasketItem.new.call(item_id)
+
+      return unless basket_item
+
+      reduce_basket_quantity(basket_item)
```

Comment:
`reduce_basket_quantity(basket_item) unless basket_item` and you can remove line 8.